final_prep/README.md

```diff
@@ -23,52 +23,58 @@ Some things to consider as you refactor include...
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+### Week 1:
+<img src="https://user-images.githubusercontent.com/99001315/156846206-d0ceeb6a-63e3-449b-a786-bf7e9be7ac0c.png" width="150" height="100">
+
+### Week 2:
+<img src="https://user-images.githubusercontent.com/99001315/156846320-de0e9bf5-e19f-42c8-9b7d-6f25c000ce83.png" width="150" height="100">
+
+### Week 3:
+<img src="https://user-images.githubusercontent.com/99001315/156846474-59a7915d-b857-43c9-a664-6a09357c21f2.png" width="150" height="100">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [✓] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+> I accidently filled out the questionare and checkmarked all the boxes for mentors before reading the instructions. Hopefully that doesn't mess up my matching with a mentor. 😳  Please let me know if I should fill it out again if it needs to be reset.
+- [✓] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [✓] Add link to your gist [here](https://gist.github.com/ruezheng/fa706909ca6e3d9686093566b9b35f69)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [✓] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [✓] Add a link to your gist [here](https://gist.github.com/ruezheng/324bf86191382d7a178e946082ec0ce4)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [✓] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [✓] Add a link to your gist [here](https://gist.github.com/ruezheng/aa35c76b0da1299c3c450d330167d444) 
```

Comment:
Rue - between this and our mentor DTR, it's clear you are in a place to have really strong and productive working relationships with peers and mentors!